grpc/weaviate.proto

```diff
@@ -110,18 +115,19 @@ message Filters {
     bool value_bool = 6;
     float value_float = 7;
     google.protobuf.Timestamp value_date = 8;
-    strArray value_str_array = 9;
-    intArray value_int_array = 10;
-    boolArray value_bool_array = 11;
-    floatArray value_float_array = 12;
-    dateArray value_date_array = 13;
+    StrArray value_str_array = 9;
```

Comment:
Can we rename the type from `StrArray` to `StringArray`? (I think str is used only in Python to describe string) or shouldn't it be here `TextArray` as we have deprecated `String` type?